### ISSUE TITLE
Client crash on console tab character

### ISSUE THREAD

**Issue by Woovie** [multitheftauto/mtasa-blue#936]
**Describe the bug**
Client crash on `<symbol><tab>` press in console. It requires some form of characters to exist before the tab press. Once you have anything and press tab, the client crashes core.

```
Version = 1.5.6-release-16927.0.000
Time = Sun May 12 19:44:52 2019
Module = C:\Program Files (x86)\MTA San Andreas 1.5\mta\core.dll
Code = 0xC0000005
Offset = 0x00024D87

EAX=13D80BA8  EBX=000F0001  ECX=20787463  EDX=20787464  ESI=20787463
EDI=000F0001  EBP=0177F9F0  ESP=0177F9A0  EIP=0F514D87  FLG=00210206
CS=0023   DS=002B  SS=002B  ES=002B   FS=0053  GS=002B
```

**To reproduce**
1. Start the game
2. Open the console
3. Type something, even a space
4. Press tab
**Version**
1.5.6-release-16927.0.000

**Comment by FileEX** [comment]
I can't reproduce

**Comment by ghost** [comment]
Reproduceable with latest custom build:

Error occuring in https://github.com/multitheftauto/mtasa-blue/blob/master/Client/core/CConsole.cpp#L369 (Access violation)

- Luxy.c

**Comment by patrikjuvonen** [comment]
Thanks for the issue! This should be fixed now in 40e09e5eb6a13b5516e0273b2efd45c894289a39.
